src/Compilers/Core/Portable/NativePdbWriter/PdbWriter.cs

```diff
@@ -956,13 +963,13 @@ private ISymUnmanagedDocumentWriter GetDocumentWriter(DebugSourceDocument docume
 
                 _documentMap.Add(document, writer);
 
-                var checksumAndAlgorithm = document.ChecksumAndAlgorithm;
-                if (!checksumAndAlgorithm.Item1.IsDefault)
+                DebugSourceInfo info = document.GetSourceInfo();
+                if (!info.Checksum.IsDefault)
                 {
                     try
                     {
-                        var algorithmId = checksumAndAlgorithm.Item2;
-                        var checksum = checksumAndAlgorithm.Item1.ToArray();
+                        var algorithmId = info.ChecksumAlgorithmId;
+                        var checksum = info.Checksum.ToArray();
```

Comment:
Will do this when native PDB support is added, now it would just be unrelated change with native PDB support reverted.